src/time_zone_lookup.cc

```diff
@@ -207,8 +212,9 @@ time_zone local_time_zone() {
   char* localtime_env = nullptr;
   if (strcmp(zone, "localtime") == 0) {
 #if defined(_MSC_VER)
-    // System-specific default is just "localtime".
-    _dupenv_s(&localtime_env, nullptr, "LOCALTIME");
```

Comment:
Using `${LOCALTIME}` to override the name _"localtime"_ is part of the API, and cannot be removed.